### DISCUSSION TITLE
Using UFW on host

### DISCUSSION THREAD

**Discussion by the-voidl** [nginx-proxy/nginx-proxy#1905] [General]
Recently I learned about this know issue that docker bypasses ufw.
I tried a lot  of workarounds, but though I don't want to break my whole configuration I'm a bit scared of playing around with the networks. Also most of the approaches only work for single containers...

The actual **Problem**: ufw blocks on the host system don't apply to my docker containers. I can still reach all websites from a blocked IP.

Is there any recommended way to enable ufw on the host machine for `proxy-companion`?

I used the prewritten docker-compose file from nextcloud. They define a network "proxy-tier" for the proxy. In this repo I can't find anything like this. Maybe this is the problem?

**Comment by buchdag** [comment]
Hi. This question concerns `nginx-proxy` which does the actual proxying, not `letsencrypt-ningx-proxy-companion`. The companion only concern is SSL certificate creation.

**Comment by tkw1536** [comment]
This sounds like an issue with ufw and docker, not nginx-proxy. 
See https://github.com/docker/for-linux/issues/690.

**Comment by the-voidl** [comment]
Oh thanks for transferring.
Idk in how far this is a docker issue, since I found posts about this back from 2015 and it seems that the docker team won't fix it.

So I woundered if there is a way to include it in nginx-proxy.
Usually, there is only one net exposed to the outer world: This one the proxy creates. In my eyes it would be easies to route this network over ufw or whatever host-system firewall.
Correct me if I'm wrong
